### ISSUE TITLE
compute_euclidean_distance

### ISSUE THREAD

**Issue by FishermanZzhang** [tiagofrepereira2012/examples.tensorflow#1]
'''
 d = tf.sqrt(tf.reduce_sum(d, axis=1)) 
'''
